packages/atomic/src/components/common/template-system/cell-desktop.pcss

```diff
@@ -211,6 +212,13 @@
       width: 100%;
     }
 
+    $(section-children) {
```

Comment:
Most of these new rules added in (cell-desktop/cell-mobile/row-desktop/row-mobile) etc. should not be added in the generic `common/temnplate-system` folder, but instead in the commerce specific `commerce/atomic-product/atomic-product.pcss`.

The reason is that all these new rules are there for `product-child`, which is not a generic / use case agnostic component.